core/src/main/java/fr/sncf/osrd/api/pathfinding/PathfindingResultConverter.java

```diff
@@ -41,9 +40,11 @@ public static PathfindingResult convert(
         var res = new PathfindingResult(pathLength);
 
         // Builds a mapping between routes and all user defined waypoints on the route
-        var userDefinedWaypointsPerRoute = HashMultimap.<SignalingRoute, Double>create();
```

Comment:
Is there a reason to change from the multimap to a map of list?

I don't really care either way, but any change here will cause extra conflicts with the other branch

Reply:
Yes, if we have two points on the same SignalingRoute and with the same offset. The multimap will merge the waypoints

Reply:
Is it the expected behavior? It would mean that if the user places several waypoints at the same location, there would only be one step. In the api version that would cause a crash, but I don't know about the editoast version.